lighthouse-core/lib/network-recorder.js

```diff
@@ -231,7 +231,6 @@ class NetworkRecorder extends EventEmitter {
     const modifiedData = {
       ...data,
       // Copy over the initiator as well to match DevTools behavior
-      // TODO(phulce): abandon this DT hack and update Lantern graph to handle it
```

Comment:
I'm not convinced this is necessary anymore, it's probably easier to just stay in sync with wht devtools does